tests/ztest/base/prj.conf

```diff
@@ -0,0 +1 @@
+CONFIG_ZTEST=y
```

Comment:
Since `testcase.yaml` specifies the default test `type` as `unit`, it is not necessary to do this.
https://github.com/zephyrproject-rtos/zephyr/blob/d1d603afc3558f31a4728d70179d1cb5d74890c5/tests/ztest/base/testcase.yaml#L2-L4

If you want to run this manually, you need to do `cmake -DBOARD=unit_testing`.